app/src/main/java/com/mapbox/mapboxsdk/plugins/testapp/activity/scalebar/ScalebarActivity.kt

```diff
@@ -0,0 +1,65 @@
+package com.mapbox.mapboxsdk.plugins.testapp.activity.scalebar
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import com.mapbox.mapboxsdk.camera.CameraUpdateFactory
+import com.mapbox.mapboxsdk.maps.Style
+import com.mapbox.mapboxsdk.plugins.testapp.R
+import com.mapbox.pluginscalebar.ScaleBar
+import kotlinx.android.synthetic.main.activity_scalebar.*
+
+/**
+ * Activity showing a scalebar used on a MapView.
+ */
+class ScalebarActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_scalebar)
+        mapView.onCreate(savedInstanceState)
+        mapView.getMapAsync { mapboxMap ->
+            mapboxMap.setStyle(Style.MAPBOX_STREETS) {
+                mapboxMap.moveCamera(CameraUpdateFactory.zoomTo(2.0))
```

Comment:
NIT: Do we need this invocation? Initially looked to me like the state saving was broken.